thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -0,0 +1,306 @@
+<div class="manage-alert-explore">
+  {{#if (not isReplayPending)}}
+    {{!-- {{#if filteredAnomalies}} --}}
+      <div class="te-pill-selectors">
+        {{!-- Date range selector --}}
+        <div class="te-pill-selectors__label">Showing</div>
+        <ul class="te-alert-page-hzlist">
+          {{#each timeRangeOptions as |range|}}
+            <li class="te-alert-page-hzlist__item {{if range.isActive "te-alert-page-hzlist__item--active"}}" {{action "onRangeOptionClick" range}}>
+              {{range.name}}
+              {{#if (eq range.name "Custom")}}
+                : {{date-range-picker
```

Comment:
Is the ":" a typo here?

Reply:
No. "Custom" is the only time range option with a semicolon:
<img width="818" alt="screen shot 2017-12-22 at 7 15 33 pm" src="https://user-images.githubusercontent.com/11814420/34315843-9122a494-e74c-11e7-9d54-c39bdf7295fe.png">